### DISCUSSION TITLE
Looking for examples of sidekiq.cr used in production with a Rails app

### DISCUSSION THREAD

**Discussion by erlend-sh** [mperham/sidekiq.cr#118] [General]
I’m very excited by the prospect of [running Sidekiq.cr besides Sidekiq.rb to handle compute-heavy jobs](http://www.mikeperham.com/2016/06/14/test-driving-sidekiq-and-crystal/).

But I’m struggling to find any real world user accounts of this being done. Has anyone successfully sped up their application with this and, better yet, blogged about it?

**Comment by icyleaf** [comment]
I wrote one in chinese if you can read it: http://icyleaf.com/2017/04/sidekiq-works-in-ruby-togethor-with-crystal/ :)

**Comment by antarr** [comment]
Something accessing the database would be great.
